src/java.net.http/share/classes/java/net/http/package-info.java

```diff
@@ -37,8 +37,10 @@
  * </ul>
  *
  * <p> The protocol-specific requirements are defined in the
- * <a href="https://tools.ietf.org/html/rfc7540">Hypertext Transfer Protocol
- * Version 2 (HTTP/2)</a>, the <a href="https://tools.ietf.org/html/rfc2616">
+ * <a href="https://tools.ietf.org/html/rfc9114">Hypertext Transfer Protocol
+ * Version 3 (HTTP/3)</a>, the <a href="https://tools.ietf.org/html/rfc7540">
+ * Hypertext Transfer Protocol Version 2 (HTTP/2)</a>, the
+ * <a href="https://tools.ietf.org/html/rfc2616">
```

Comment:
It is strange to link to a different place here than in the `@spec` tags below. I will fix that.